multilep/src/TriggerAnalyzer.cc

```diff
@@ -84,6 +84,8 @@ TriggerAnalyzer::TriggerAnalyzer(const edm::ParameterSet& iConfig, multilep* mul
                                    "HLT_AK8PFJet500"};
 
   } else {
+    allFlags["passTrigger_mAN094"]      = {"HLT_IsoMu22", "HLT_IsoTkMu22", "HLT_IsoMu22_eta2p1", "HLT_IsoTkMu22_eta2p1"};
+
     allFlags["passTrigger_m"]   = {"HLT_IsoMu24", "HLT_IsoTkMu24", "HLT_Mu50", "HLT_TkMu50", "HLT_Mu45_eta2p1"}; // HLT_TkMu50 off for ~3/fb, HLT_Mu45_eta2p1 off for ~12/fb
```

Comment:
Even though the Muon POG seems to recommend the triggers currently listed in passTrigger_m for 2016 (see https://twiki.cern.ch/twiki/bin/view/CMS/MuonHLT2016), I guess the passTrigger_mAN094 are still good to use, there's always at least one unprescaled of them, so it should be save to use as long as these four are in OR.
Not sure why the Muon POG does not mention them though.